roottest/root/treeformula/sync/CMakeLists.txt

```diff
@@ -1,9 +1,40 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
+ROOTTEST_ADD_TEST(alias
+                  MACRO runalias.C+
+                  OUTREF alias.ref)
 
-ROOTTEST_ADD_OLDTEST()
+ROOTTEST_COMPILE_MACRO(loadcode.C
+                       FIXTURES_SETUP root-treeformula-sync-loadcode-fixture)
 
+ROOTTEST_ADD_TEST(load
+                  MACRO runload.C
+                  OUTREF load.ref
+                  FIXTURES_REQUIRED root-treeformula-sync-loadcode-fixture)
+
+ROOTTEST_ADD_TEST(ScanFail
+                  MACRO execScanFail.C
+                  COPY_TO_BUILDDIR run00155-short.root
+                  OUTREF execScanFail.ref)
+
+ROOTTEST_ADD_TEST(writefile
+                  COMMAND ${CMAKE_BINARY_DIR}/roottest/root/io/event/TestIoEvent 10 1 0 1 600
+                  FIXTURES_REQUIRED root-io-event-TestIoEvent-fixture
+                  FIXTURES_SETUP root-treeformula-sync-writefile-fixture)
+
+ROOTTEST_ADD_TEST(readfile-0
+                  MACRO sync.C
+                  MACROARG 0
+                  ROOTEXE_OPTS -e "gInterpreter->AddIncludePath(\"-I${CMAKE_SOURCE_DIR}/roottest/root/io/event\")"
+                               -e "gSystem->AddDynamicPath(\"${CMAKE_BINARY_DIR}/roottest/root/io/event\")"
+                               -e "(void) gSystem->Load(\"libTestIoEvent\")"
+                  OUTREF readfile-0.ref
+                  WILLFAIL
```

Comment:
The test does fail but this is (if I recall correctly) a bug and thus not a desirable failure.  We should open a github issue to record that this needs investigation.

Reply:
In Makefile `Run.C(0)` was executed only when failing tests were selected - `ifneq ($(FAIL),)`
Here is issue: https://github.com/root-project/root/issues/19238

Reply:
Thanks.